test_history.py

```diff
@@ -114,3 +115,13 @@ def test_protected_key():
                   History.RESERVED_KEY_KEY)
     assert_raises(ValueError, h.__setitem__,
                   History.RESERVED_KEY_KEY, 'aardvark')
+
+
+def test_repr():
+    h.clear()
+    kvpairs = (('foo', 'bar'), ('spam', 'spam spam spam'))
+    dct = {}
+    for k, v in kvpairs:
+        dct[k] = v
+        h[k] = v
+    assert repr(h) == repr(dct)
```

Comment:
FYI, `assert_equal` gives a better error message in the event of failure, but probably not worth going back for that.